lib/elixir/pages/Writing Documentation.md

```diff
@@ -74,17 +74,19 @@ Code comments are for developers reading the code. They are useful to mark impro
 
 In other words, documentation is required, code comments are optional.
 
-## Privacy
+## Hiding Internal Modules and Functions
 
-Elixir does not allow developers to document private functions. That's because private functions only exist inside the module that defines them and cannot be accessed externally.
+Libraries often have two levels of public interface: on one hand, they expose an API for external clients, but on the other hand, the modules and functions that conform the library often need to use other modules and functions in the library which are not part of the external public interface.
+
```

Comment:
Also, maybe we can squash the previous and next paragraphs together, wdyt? I feel like they're saying very correlated things :)